### ISSUE TITLE
Album not created when a new track is added via `watchdoge.py`

### ISSUE THREAD

**Issue by geoffrey45** [geoffrey45/alice#44]
A single track object is created but the album object is not created:

```py
File "/home/cwilvx/code/alice/server/app/api/album.py", line 45, in get_album_tracks
    album = api.ALBUMS[index]
TypeError: list indices must be integers or slices, not NoneType

```

**Comment by geoffrey45** [comment]
Might be fixed by #70

**Comment by geoffrey45** [comment]
fixed in another PR
